recipes/pysmithchart/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "pysmithchart" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pysmithchart-{{ version }}.tar.gz
+  sha256: 9b10439a9b0e9883fac29308bfb5d78e18fca978197f52d5bc19efb436ef1cc6
+
+build:
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv"
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python {{ python_min }}
+    - setuptools
+  run:
+    - matplotlib-base
+    - numpy
+    - python {{ python_min }}
+    - scipy
+    - pytest
+
+test:
+  imports:
+    - pysmithchart
+  requires:
+    - pytest
+    - pip
+  source_files:
+    - tests/test_simple.py
```

Comment:
The test files aren't in the source distribution on pypi, so this should be removed along with the `pytest` stuff

Reply:
that was my first thought, but i wanted to try to add the test file back in.  If it does not work on the first try, I will follow your advice!

Reply:
Also, last time I tried to add a repo to conda-forge it was weeks before anyone could get around to committing the port once it had build cleanly.  It is now very nice to have timely feedback!!